### PR TITLE
Create tasks using minimal dict representation

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**
- creates tasks from minimal dict representations (see `TestClient.test_enqueue_new_task_dict`)
- preserves compatibility with existing app logic

**Note**: these changes are a prerequisite for upcoming RPC updates (i.e. task testing)

### CODE REVIEW

File: apps/blender/task/blenderrendertask.py [hunk 1]
```diff
@@ -141,16 +141,25 @@ def get_task_border(cls, subtask, definition, total_subtasks,
...
             parts = int(total_subtasks / frames)
-            return cls.__get_border((start_task - 1) % parts + 1, (end_task - 1) % parts + 1,
+            return cls.__get_border((start_task - 1) % parts + 1,
+                                    (end_task - 1) % parts + 1,
                                     parts, res_x, res_y)
 
         return []
 
     @classmethod
+    def get_task_borders(cls, definition, total_subtasks):
```
Comment (badb) [inline apps/blender/task/blenderrendertask.py]: What is the connection between this method and `get_task_border`?

File: golem/client.py [hunk 2]
```diff
@@ -334,19 +334,24 @@ def stop_network(self):
         self.nodes_manager_client = None
 
     def enqueue_new_task(self, task):
+        if isinstance(task, dict):
```
Comment (badb) [inline golem/client.py]: Can we add some #FIXME to this part, to remember to fix it after new interface is merged?

**Review by badb — APPROVED** [review]

File: golem/client.py (hunk 2 above)
Comment (mfranciszkiewicz) [reply golem/client.py]: Done

File: apps/blender/task/blenderrendertask.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply apps/blender/task/blenderrendertask.py]: Please see the updated border generation code

**Pull request merged**